src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Header.component.tsx

```diff
@@ -53,28 +54,33 @@ const StyledTableCell = withStyles(() =>
  */
 export function TableContentHeaderComponent(
     isSumRowCol: boolean,
-    headerValues: string[]
+    headerValues: string[], 
+    tableOption: DisplayOptionType,
 ): JSX.Element[] {
     const { t } = useTranslation(["QueryPage"]);
-    const elements: JSX.Element[] = [];
-    elements.push(
+    const headerTableCells: JSX.Element[] = [];
+    headerTableCells.push(
         <StyledTableCell key="header-blank" css={blankCellStyle}>
             &nbsp;
         </StyledTableCell>
     );
-    headerValues.forEach((element): void => {
-        elements.push(
-            <StyledTableCell key={`header-${element}`} css={headerCellStyle}>
-                {element}
+    headerValues.forEach((headerValue): void => {
+        let headerTitle = headerValue
+        if (tableOption === "relative") {
+            headerTitle = `${headerValue} in %` 

```

Comment:
I would use a translation here too, even if it is the same text for now.

Reply:
Done.